### ISSUE TITLE
Outfits Class

### ISSUE THREAD

**Issue by AnthonyMed13** [uprm-inso4115-2023-2024-s2/semester-project-photofit-a-closet-inventory-app#113]
Comment describing this issue down below.

**Comment by HectorSotelo** [comment]
Outfits are formed from a bundle of clothes, 3 essentials, Shirt/UpperBody Wear, Pants/Skirts/LowerBody Wear, and Shoes/Foot Wear. 
Outfits can have missing clothing items, such as no shirt (beach/pool outfit), no pants/skirt (dress is upperbody wear), and no shoes (beach/pool/meditation outfit)
Outfits must have a name.

Deliverable: An entity that stores its own identification and the identification of only one upperbody wear, only one lowerbody wear, and only one footwear.

**Comment by Adhel-Esteban1** [comment]
I started implementation on the Outfits class because I needed it for my styling issue #120 Seamless design. I managed to implement a seemingly "working" Outfits class, and it can be called from certain areas. However, the other logic to edit the class and add to the DB is still to be implemented. But I will assign myself to this because I worked part of it. But we still may need another developer if you see something else that is needed.
